client/java/src/main/java/io/openlineage/client/transports/CompositeConfig.java

```diff
@@ -28,13 +28,13 @@ public final class CompositeConfig implements TransportConfig, MergeConfig<Compo
 
   @Getter @Setter private List<TransportConfig> transports;
 
-  @Getter @Setter private boolean continueOnFailure;
+  @Getter @Setter private Boolean continueOnFailure;
```

Comment:
why the need to have boxed values?

Reply:
If someone doesn't pass the value in the configuration it defaults to false. We want to be able to tell if the value was empty (and we should replace it with `true`)